mytest/models.py

```diff
@@ -44,6 +44,19 @@ def archive(self, *args, **kwargs) -> None:
         pass
 
 
+class SubFakeModel(FreezableFSMModelMixin):
+    FROZEN_IN_STATES = (
+        FakeStates.ACTIVE.value,
+        FakeStates.ARCHIVED.value,
+    )
+    FSM_STATE_FIELD_NAME = 'fake_model.state'
```

Comment:
maybe it should be
```suggestion
    FSM_STATE_FIELD = FakeModel.state
```

Then we could skip duplicate FROZEN_IN_STATES.

Reply:
I will try

Reply:
https://github.com/ming-tung/django-fsm-freeze/pull/8 goes into that direction. I might be able to do it once it's accepted.